sys/fs/fuse/fuse_vnops.c

```diff
@@ -2774,11 +2779,12 @@ fuse_vnop_setextattr(struct vop_setextattr_args *ap)
 
 	if (err == ENOSYS) {
 		fsess_set_notimpl(mp, FUSE_SETXATTR);
-		err = EOPNOTSUPP;
+		err = EXTERROR(EOPNOTSUPP, "This daemon does not implement "
+		    "setting extattrs");
 	}
 	if (err == ERESTART) {
 		/* Can't restart after calling uiomove */
-		err = EINTR;
+		err = EXTERROR(EINTR, "Operation was interrupted");
```

Comment:
EINTR is self explanatory.  I don't think the extended error message adds any value.